plugins/draw-tools.js

```diff
@@ -676,16 +719,91 @@ window.plugin.drawTools.boot = function() {
     runHooks('pluginDrawTools',{event:'layersEdited'});
   });
+
   //add options menu
   $('#toolbox').append('<a onclick="window.plugin.drawTools.manualOpt();return false;" accesskey="x" title="[x]">DrawTools Opt</a>');
 
   $('head').append('<style>' +
         '.drawtoolsSetbox > a { display:block; color:#ffce00; border:1px solid #ffce00; padding:3px 0; margin:10px auto; width:80%; text-align:center; background:rgba(8,48,78,.9); }'+
-        '.ui-dialog-drawtoolsSet-copy textarea { width:96%; height:250px; resize:vertical; }'+
+        '.ui-dialog-drawtoolsSet-copy textarea { width:96%; height:150px; resize:vertical; }'+
         '</style>');
 
 }
 
+// ---------------------------------------------------------------------------------
+// EXPORT (POLYS AS LINES)
+// ---------------------------------------------------------------------------------
+window.plugin.drawTools.getDrawAsLines = function(){
+  var rawDraw = JSON.parse(window.localStorage[window.plugin.drawTools.KEY_STORAGE]);
+  var draw = [];
+
+  for (i in rawDraw){
+    var elemDraw = rawDraw[i];
+
+    if (elemDraw.type === 'polygon'){
+      var convElemDraw = {};
+      convElemDraw.color = elemDraw.color;
+      convElemDraw.type = 'polyline';
+      convElemDraw.latLngs = [];
+
+      for (j in elemDraw.latLngs){
+        var ll = elemDraw.latLngs[j];
+        convElemDraw.latLngs.push(ll);
+      }
+      convElemDraw.latLngs.push(elemDraw.latLngs[0]);
+
+      draw.push(convElemDraw);
+    } else {
+      draw.push(elemDraw);
+    }
+  }
+
+  return JSON.stringify(draw);
+};
+
+// ---------------------------------------------------------------------------------
+// EMPTY POLYGONS (EMPTY DRAWN FIELDS)
+// ---------------------------------------------------------------------------------
+window.plugin.drawTools.EDFstatus = false;
+
+window.plugin.drawTools.initEDF = function(){
+  // window.addHook('pluginDrawTools', window.plugin.drawTools.edf.hookManagement);
+  window.addHook('iitcLoaded', function(){
+    if (window.plugin.drawTools.EDFstatus){
+      window.plugin.drawTools.toggleOpacityOpt();
+      window.plugin.drawTools.clearAndDraw();
+    }
+  });
+};
+
+window.plugin.drawTools.toggleEDF = function(){
+  var status = window.plugin.drawTools.EDFstatus;
+  status = Boolean(!status);
+  window.plugin.drawTools.EDFstatus = status;
+};
+
+window.plugin.drawTools.toggleOpacityOpt = function(){
+  if (window.plugin.drawTools.EDFstatus){
+    window.plugin.drawTools.polygonOptions.fillOpacity = 0.0;
+    window.plugin.drawTools.polygonOptions.interactive = false;
+  } else {
+    window.plugin.drawTools.polygonOptions.fillOpacity = 0.2;
+    window.plugin.drawTools.polygonOptions.interactive = true;
+  }
+};
+
+window.plugin.drawTools.clearAndDraw = function(){
+  window.plugin.drawTools.drawnItems.clearLayers();
+  window.plugin.drawTools.load();
+  console.log('DRAWTOOLS: reset all drawn items');
+};
+
+window.plugin.drawTools.edfStausToggle = function(){
```

Comment:
is it `edfStatusToggle` ? (same for the calls)

Reply:
done